src/graphing/radar.js

```diff
@@ -828,12 +828,30 @@ const Radar = function (size, radar) {
     })
 
     if (featureToggles.UIRefresh2022) {
-      renderRadarLegends(radarElement)
+      let hasMovements = hasMovementData(quadrants)
+      renderRadarLegends(radarElement, hasMovements)
       hideTooltipOnScroll(tip)
       addRadarLinkInPdfView()
     }
   }
 
+  function hasMovementData(quadrants) {
+    let hasMovements = false
+
+    for (var quadrantWrapper of quadrants) {
+      let quadrant = quadrantWrapper.quadrant
+
+      for (var blip of quadrant.blips()) {
+        if (blip.status() !== '') {
+          hasMovements = true
```

Comment:
Could we just `return true` here instead (and then `return false` instead of `return hasMovements`) to get rid of the `hasMovements` variable?

Reply:
Deal 🤝